x-pack/plugins/cases/common/types/api/observable/v1.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+import { limitedArraySchema, NonEmptyString } from '../../../schema';
+import { CaseObservableRt, ObservablePatch } from '../../domain';
+
+/**
+ * Observables
+ */
+
+const MIN_DELETE_IDS = 1;
+const MAX_DELETE_IDS = 10;
+
+export const BulkDeleteObservablesRequestRt = rt.strict({
+  ids: limitedArraySchema({
+    codec: NonEmptyString,
+    min: MIN_DELETE_IDS,
+    max: MAX_DELETE_IDS,
+    fieldName: 'ids',
+  }),
+});
+
+export type BulkDeleteObservablesRequest = rt.TypeOf<typeof BulkDeleteObservablesRequestRt>;
+
+export const ObservableRequestRt = rt.strict({
+  observables: rt.array(ObservablePatch),
```

Comment:
I think we should support only adding one observable at the time at the moment. Also, the version is not needed. See my comment on the route.